### ISSUE TITLE
delete temporary files after creating final file from thumbnail upload

### ISSUE THREAD

**Issue by tobiasschweizer** [dasch-swiss/sipi#182]
When running the integration test in Knora, temporary files remain in the folders `tmp` and `thumbs`. I think they should be deleted when callling `convert_from_file.lua`.

**Comment by subotic** [comment]
Yes, they should be deleted as there is code that should do just that. I will check.

**Comment by tobiasschweizer** [comment]
Thanks!
